### PR TITLE
chore: specify ts lib folder path

### PR DESCRIPTION
**Pull request by nguyentoanit**

### CODE REVIEW

**Comment by nguyentoanit** [comment]
@moltar : This bug is quite strange. 
Each attempt has a different result. ([Attempt 1](https://github.com/ScaleLeap/selling-partner-api-sdk/runs/2489309823?check_suite_focus=true), [Attempt 2](https://github.com/ScaleLeap/selling-partner-api-sdk/runs/2489354294?check_suite_focus=true), [Attempt 3](https://github.com/ScaleLeap/selling-partner-api-sdk/runs/2489344318?check_suite_focus=true))
😞 

There was [the success message](https://github.com/nguyentoanit/selling-partner-api-sdk/runs/2490383371?check_suite_focus=true#step:6:196) when generating. But [API model directory](https://github.com/nguyentoanit/selling-partner-api-sdk/runs/2490383371?check_suite_focus=true#step:6:255) was still unavailable.

At first, I thought rate limit might be cause. But after try with url has 429 status, `openapi-generator-cli` will throw an invalid url error.

Any suggestions for me? 🤔

**Comment by moltar** [comment]
some sort of race condition?

**Comment by nguyentoanit** [comment]
This PR fixes the issues: 
- Enum/union goes back: https://github.com/ScaleLeap/selling-partner-api-sdk/pull/89#issuecomment-831011595

- Missing api model error: https://github.com/ScaleLeap/selling-partner-api-sdk/runs/2489354294?check_suite_focus=true

PR:
https://github.com/ScaleLeap/selling-partner-api-sdk/pull/92


Action log:
https://github.com/ScaleLeap/selling-partner-api-sdk/actions/runs/808921576/workflow

**Review by moltar — APPROVED** [review]

File: .github/workflows/generate-models.yml [hunk 1]
```diff
@@ -20,6 +20,12 @@ jobs:
         with:
           node-version: 14
 
+      - name: Setup Java
```
[the remark this answers is not in the document]
Comment (nguyentoanit) [reply .github/workflows/generate-models.yml:23]: I realized that [error handling in executor](https://github.com/ScaleLeap/selling-partner-api-sdk/blob/master/utils/generator/api-model-generator.ts#L38) might not be working correctly. 

Although, Java is missing but it didn't throw an error.
I'm going to investigate them today.

File: .github/workflows/generate-models.yml [hunk 2]
```diff
@@ -20,6 +20,12 @@ jobs:
...
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
```
[the remark this answers is not in the document]
Comment (nguyentoanit) [reply .github/workflows/generate-models.yml:26]: I will fix in the next PR. Thanks.

**Pull request merged**